packages/client/package.json

```diff
@@ -149,6 +149,7 @@
     }
   },
   "dependencies": {
+    "@codspeed/benchmark.js-plugin": "1.0.1",
```

Comment:
somehow still in `dependencies` although the commit message says it was moved

Reply:
This is what it looks like when I mess up :D